routes/OTHERpostRoutes.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../database/models");
+require("dotenv").config();
+
+const post = db.Post;
+
+router.post('/add', async(req, res)=>{
+    console.log(req.body)
+    const { color, size, location, petName, email, phoneNumber, PostTypeId } = req.body;
+    const { mimetype, originalname, buffer } = req.file;
+
+    const newPost = await post.create({
+        color,
+        size,
+        location,
+        petName,
+        email,
+        phoneNumber,
+        PostTypeId,
+        StatusId: 2,
+        imageType: mimetype,
+        imageName: originalname,
+        imageData: buffer
```

Comment:
I added what I hope is the ability to store images in our DB. However, I don't think I can test it with postman so we will have to wait for Ben to finish the form and I will test with the form/make any changes that I need to. Using something called [Multer](https://www.npmjs.com/package/multer).